commit 3a4ab4ddbf903b77560317c2525470c359a97a33 (refs/changes/80/4780/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-09-13T16:46:32+02:00 (1 year, 1 month ago)
    
    TRITON-686 workflow needs to support image.requirements.bootrom
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/package.json b/package.json
index 71ff137..4305e6d 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "workflow",
   "description": "SmartDataCenter Workflow API and Runners",
-  "version": "1.3.2",
+  "version": "1.3.3",
   "author": "Joyent (joyent.com)",
   "private": true,
   "repository": {
@@ -12,7 +12,7 @@
     "async": "0.1.22",
     "bunyan": "1.8.1",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
-    "imgmanifest": "git+https://github.com/joyent/node-imgmanifest.git#e21ea53",
+    "imgmanifest": "3.1.0",
     "lstream": "0.0.4",
     "restify": "4.1.1",
     "sdc-clients": "11.3.1",
